requirements.txt

```diff
@@ -7,3 +7,4 @@ ply>=3.10
 psutil>=5
 scipy>=1.0
 sympy>=1.3
+seaborn>=0.9.0
```

Comment:
This should be moved to the dev requirements.  Then it should be added to the `setup.py` file under the `extra_requires` visualization part.  Then it is only needed when requesting matplotlib functionality.

Reply:
I'm wondering why this is needed at all. It looks like we're only using this for a colormap. Why can't we just use a colormap builtin to matplotlib? Or if none of the builtins are good enough just construct one without needed to add a new requirement. 

I think most people end up installing the visualization packages and adding an extra requirement to that list just to get a colormap seems a bit much. I think it would be a different story if we were using seaborn for it's integration with pandas or other features that are harder to get than with just mpl.

Reply:
Well mainly because they supply proper angular color maps that are also a bit more lively than the mpl `twilight` ones.  There are likely other uses for seaborn color maps in other routines as well.  So I would say it is not just for this.

As a personal opinion, I find it hard  to imagine people will complain about a 200kb package that is used by 25k other packages on github.

Reply:
Well people complain whenever we add any dependency regardless of how small or commonly used it is. I think it's partly because some of users operate in extremely locked down environments where installing any new software requires IT approval. But sure, that makes sense to me.